packages/contentful--app-scripts/lib/utils.js

```diff
@@ -81,7 +100,31 @@ function getActionsManifest() {
   ----------------------------`);
     console.log('');
 
-    return manifest.actions.map((action) => ({ parameters: [], ...action })); // adding required parameters
+    const actions = manifest.actions.map((action) => {
+      const allowedNetworks = Array.isArray(action.allowedNetworks) ? action.allowedNetworks : [];
+
+      const hasInvalidNetworks = allowedNetworks
+        .map(removeProtocolFromUrl)
+        .some((netWork) => !isValidNetwork(netWork));
```

Comment:
I would replace `some` with `find` to be able to get exact invalid network and show it in error message below. Same computing price, but we can give a bit more info to the user that way.